sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/MessageUtils.java

```diff
@@ -253,4 +275,75 @@ private static TransactionalState getTransactionState(ByteBuffer transactionId,
         transactionalState.setOutcome(outcome);
         return transactionalState;
     }
+
+    public static ServiceBusMessage traceMessageSpan(ServiceBusMessage serviceBusMessage,
+        Context messageContext, String hostname, String entityPath, TracerProvider tracerProvider) {
```

Comment:
How does Kind fits into this ?\, should that be passed here ?

https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/core/azure-core/src/main/java/com/azure/core/util/tracing/ProcessKind.java
@samvaity    Can you look into this PR ?

Reply:
`ProcessKind.MESSAGE` should be used to create a message. Just like in EventHubs.

Reply:
The ProcessKind helps the tracer apply specific attributes on the span. For example, in the case of MESSAGE, the tracer will attribute the spanType=Producer, [here](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/core/azure-core-tracing-opentelemetry/src/main/java/com/azure/core/tracing/opentelemetry/OpenTelemetryTracer.java#L87-L90)

Reply:
@samvaity is this javadoc misleading? It looks like MESSAGE is used for receiving.
```java
    /**
     * Amqp message process call to receive data.
     */
    MESSAGE,
```

Reply:
Yeah, I think we initially had this as RECEIVE and later updated to MESSAGE so the left over javadoc. But yes it should be updated to suit process kind "message". AMQP process kind message